docs/cubeviz/plugins.rst

```diff
@@ -335,6 +335,19 @@ have valid flux units. For 3D data, the current :ref:`slice` is used.
     :ref:`Imviz Aperture Photometry <aper-phot-simple>`
         Imviz documentation describing the concept of aperture photometry in Jdaviz.
 
+.. _cubeviz-sonify-data:
+
+Sonify Data
+===========
+
+This plugin uses the `Strauss <https://strauss.readthedocs.io/en/latest/>`_ package
```

Comment:
Should add a note to users on how to install the optional dependencies and maybe even mention the non-Python dependency of libportaudio2 (is that a Linux only thing?).

Reply:
in STRAUSS we throw a [custom error](https://github.com/james-trayford/strauss/blob/ddaca6b823f774305c6520ef9afd714c48cea7ac/src/strauss/sonification.py#L427C1-L434C64) if you try and use the PortAudio functionality and it's not there. This seems to be [a non mac / windows problem](https://python-sounddevice.readthedocs.io/en/0.4.7/installation.html#:~:text=If%20you%20install%20the%20sounddevice%20module%20with%20pip%20on%20macOS%20or%20Windows%2C%20the%20PortAudio%20library%20(with%20ASIO%20support%20on%20Windows)%20will%20be%20installed%20automagically.%20On%20other%20platforms%2C%20you%20might%20have%20to%20install%20PortAudio%20with%20your%20package%20manager%20(the%20package%20might%20be%20called%20libportaudio2%20or%20similar).)

Reply:
I think that depends if users need to install that manually after running `pip install .[strauss]`. I am able to get all dependencies installed with that command on windows.

Reply:
At the very least maybe repeat the `[strauss]` command here? I see you already have it in the plugin doc on `.vue`.